lightning-invoice/src/utils.rs

```diff
@@ -62,9 +62,17 @@ pub fn create_phantom_invoice<Signer: Sign, K: Deref>(
 	if let Some(amt) = amt_msat {
 		invoice = invoice.amount_milli_satoshis(amt);
 	}
+	let mut route_hints: HashMap<PublicKey, (u64, RouteHint)> = HashMap::new();
```

Comment:
For phantoms, this should be per-"real"-node, not across the full outer loop (this will make it more similar to the non-phantom case as well).

Reply:
Thanks!